docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -270,6 +278,10 @@ type MachineDeploymentClass struct {
   // Template is a local struct containing a collection of templates for creation of
   // MachineDeployment objects representing a set of worker nodes.
   Template MachineDeploymentClassTemplate `json:"template"`
+  
+  // MachineHealthCheckClass defines a MachineHealthCheck for this MachineDeploymentClass.
```

Comment:
What was the resolution of the following?
> Could we move this definition under WorkersClass? And specify that this one will be applied to each one of the MachineDeployment classes unless they supply their own configuration/overrides?
> https://github.com/kubernetes-sigs/cluster-api/pull/5511#discussion_r766768674?

(Although I think @vincepri meant to copy it there so there is `machineHealthCheck` on workers and on the MD level)